include/lsst/afw/math/warpExposure.h

```diff
@@ -461,6 +461,30 @@ int warpImage(DestImageT &destImage,                            ///< remapped %i
               );
 
 /**
+ * @brief A variant of warpImage that uses a Transform<Point2Endpoint, Point2Endpoint>
+ * instead of a pair of WCS to describe the transformation.
+ *
+ * @param[in,out] destImage  Destination image; all pixels are set
+ * @param[in] srcImage  Source image
+ * @param[in] srcFromDest  Transformation from destination to source parent pixel position
+ *    This can be computed as srcWcs.getInverse().of(destWcs)
```

Comment:
I'm afraid that's how it's done. Warping requires a transformation from destination pixel position to source. I could name it destToSrc but the same math is still required to derive it. Once our old `image.Wcs` class is replaced with `geom.SkyWcs` you can call the override of warpImage that takes the source and destination WCS directly, or call `warpExposure`, and not have to think about it at all.